### DISCUSSION TITLE
Hamburger menu not working

### DISCUSSION THREAD

**Discussion by kaanaksit** [alshedivat/al-folio#781] [Q&A] [answered]
# Bug
First of all, thank you for making al-folio public. It is a great project and I am a happy user. On a browser running on an Android, my al-folio based website does not have a working hamburger menu (also known as a navigation menu collapsed into three lines). I haven't modified al-folio and using the version in your master (I believe), the source of my page is public and reachable [here](https://github.com/kunguz/kunguz.github.io). I appreciate it if you can guide me on how to fix this problem. Thanks!

# Reproduce
- Visit [my website](https://kaanaksit.com).
- Click on the hamburger menu on the right top of the page.
- Nothing happens.
- I can reproduce this on different mobile phones.

**Comment by rohandebsarkar** [comment, marked as answer]
After a preliminary look at the console logs of your site, it seems that the issue is SRI mismatch. The Bootstrap script is not loading because of the SRI mismatch. It looks like you reverted back to different scripts for bootstrap and pooper, while our version is using the bootstrap bundle (popper included). If you want to keep it that way, just fix the bootstrap js integrity in `_config.yml`

**Reply by kaanaksit** [reply]
Thank you, Rohan Debsarkar! This suggestion resolved my issue. My website cooks the hamburger right way now :) Thanks!

**Reply by dvats** [reply]
Sorry to barge in on this older conversation, but could you explain what exactly to change in the bootstrap js integrity? I have the same issue, and below is my bootstrap info in `_config.yml`:

bootstrap:
  version: "4.6.1"
  integrity:
    css: "sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY="
    js: "sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8="
